src/site/xdoc/checks/misc/translation.xml

```diff
@@ -115,32 +115,59 @@ messages.properties: Key 'ok' missing.
           extensions will be considered as files that belong to different resource bundles.
         </p>
         <p id="Example1-config">
-          To configure the check to check only files which have '.properties' and '.translations'
-          extensions:
+          To configure the check to check existence of Japanese and French translations:
         </p>
         <source>
 &lt;module name=&quot;Checker&quot;&gt;
   &lt;module name=&quot;Translation&quot;&gt;
-    &lt;property name=&quot;fileExtensions&quot; value=&quot;properties, translations&quot;/&gt;
-    &lt;property name=&quot;requiredTranslations&quot; value=&quot;fr&quot;/&gt;
+    &lt;property name=&quot;requiredTranslations&quot; value=&quot;ja, fr&quot;/&gt;
   &lt;/module&gt;
 &lt;/module&gt;
         </source>
-        <p id="Example1-code">
-          Example:
+        <p id="Example1-raw">
+          Example1:
+          (file-structure):
+        </p>
+        <source>
+--&gt; Example1
+  |--&gt; messages.properties
```

Comment:
Please make whole source block like:
```
messages.properties
messages_ja.properties
messages_fr.properties
```
Flat is good, user should not kare what is parent folder.